mayastor-test/test_nexus.js

```diff
@@ -343,30 +346,65 @@ describe('nexus', function() {
     });
   });
 
+  /*  FIXME:
```

Comment:
This looks like something we should either file an issue for or fix before integration?

Reply:
Commented-out test removed. JIRA raised: https://mayadata.atlassian.net/browse/CAS-161